website/hw/hw8.rst

```diff
@@ -638,8 +768,8 @@ Now, from your pipeline’s page, do the following:
    of the page.
 -  Enable “Create new review apps for new pull requests automatically”
 -  Do NOT enable “Wait for CI to pass” (this refers to Heroku’s own CI,
-   and we are already using Travis for CI)
--  Enable “Destroy stale review apps automatically”
+   and we are already using GitHub Actions for CI)
+-  Enable “Destroy stale review apps automatically”.
 -  Don’t worry about the “Review apps may incur dyno and add-on
```

Comment:
This doesn't appear  anywhere

Reply:
It's easy to miss, but it appears above the "Enable Review Apps" button (I've updated the text to clarify; the previous text sounded like they would see a pop-up or something like that)

![Screenshot from 2021-05-15 10-33-21](https://user-images.githubusercontent.com/448918/118369458-1301a300-b569-11eb-98c5-6000dc8a9091.png)